guides/doc-Provisioning_Guide/topics/Cloud-GCE.adoc

```diff
@@ -62,6 +83,20 @@ To add an image, complete the following steps:
 . Optional: If the selected image supports `cloud-init`, select the *User data* check box to enable user data input.
 . Click *Submit* to save the image details.
 
+.For CLI Users
+
+To add an image, enter the following command. With the `--username` option, specify a user other than `root`, because the `root` user cannot connect to a GCE instance using SSH keys. The username must begin with a letter and consist of lowercase letters and numbers.
+
+[options="nowrap" subs="+quotes"]
+----
+# hammer compute-resource image create --name '_gce_image_name_' \
+--compute-resource '_gce_cr_' \
+--operatingsystem-id 1 \
+--architecture-id 1 \
+--uuid '_378010XXXXXXXXX_' \
```

Comment:
Is this example, with all these XXXs really necessary? Is the UUID in this format? It is usually with dashes. If this value has no relation to an actual UUID, perhaps just replace with _UUID_

Reply:
@ kgaikwad, can you clarify if the uuid in this example is a real value or a dummy value?

Reply:
I have updated the real UUID with XXXs. You can replace XXXs with some numbers.

Reply:
@kgaikwad, are you able to provide a real UUID from GCE? Using a real value is always better in docs than a dummy value of XXXs. Thank you.

Reply:
@spetrosi, real UUID from GCE  - `3780108136525169178`